axelor-account/src/main/java/com/axelor/apps/account/service/config/AccountConfigService.java

```diff
@@ -38,12 +41,25 @@
 import com.axelor.i18n.I18n;
 import com.google.inject.Inject;
 import com.google.inject.persist.Transactional;
+import java.util.ArrayList;
 import java.util.List;
+import org.apache.commons.collections.CollectionUtils;
 
 public class AccountConfigService {
```

Comment:
This is not the correct service to add these methods. They must be moved to a new service